stormpath/build.gradle

```diff
@@ -0,0 +1,123 @@
+apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'com.jfrog.bintray'
+
+// This is the library version used when deploying the artifact
+version = '0.1.0'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion '23.0.2'
+
+    defaultConfig {
+        minSdkVersion 14
+        targetSdkVersion 23
+        versionName version
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+    testOptions {
+        unitTests.returnDefaultValues = true
+    }
+    lintOptions {
+        // Fix for lint error:
+        // Invalid package reference in library; not included in Android: java.nio.file. Referenced from okio.Okio.
+        warning 'InvalidPackage'
+    }
+}
+
+def siteUrl = 'https://github.com/stormpath/stormpath-sdk-android'      // Homepage URL of the library
+def gitUrl = 'https://github.com/stormpath/stormpath-sdk-android.git'   // Git repository URL
+group = 'com.stormpath.sdk'                                             // Maven Group ID for the artifact
+
+configurations {
+    javadocDeps
+}
+
+dependencies {
+    compile 'com.squareup.okhttp3:okhttp:3.1.2'
+    compile 'com.squareup.moshi:moshi:1.1.0'
+    compile 'com.android.support:support-annotations:23.1.1'
+    javadocDeps 'com.squareup.okhttp3:okhttp:3.1.2'
+    javadocDeps 'com.squareup.moshi:moshi:1.1.0'
+    javadocDeps 'com.android.support:support-annotations:23.1.1'
+    testCompile 'junit:junit:4.12'
+    testCompile 'com.squareup.assertj:assertj-android:1.1.1'
+    testCompile 'com.squareup.okhttp3:mockwebserver:3.1.2'
+}
+
+install {
+    repositories.mavenInstaller {
+        // This generates POM.xml with proper parameters
+        pom {
+            project {
+                packaging 'aar'
+
+                // Add your description here
+                name 'Stormpath Android SDK'
+                url siteUrl
+
+                // Set your license
+                licenses {
+                    license {
+                        name 'The Apache Software License, Version 2.0'
+                        url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+                developers {
+                    developer {
```

Comment:
Added in https://github.com/stormpath/stormpath-sdk-android/commit/322baf4f9b0b596e61d2c23799abc727d13be6a5.